examples/hacker-news/client/middlewares/index.js

```diff
@@ -1,40 +1,74 @@
+import { actionTypes } from 'resolve-redux'
+import { routerActions } from 'react-router-redux'
+
 import {
   OPTIMISTIC_STORY_UPVOTED,
   OPTIMISTIC_STORY_UNVOTED
 } from '../actions/actionTypes'
 import { rootDirectory } from '../constants'
-import { actionTypes } from 'resolve-redux'
-const { SEND_COMMAND } = actionTypes
+
+const {
+  SEND_COMMAND_SUCCESS,
+  SEND_COMMAND_FAILURE,
+  LOAD_VIEWMODEL_STATE_SUCCESS,
+  LOAD_READMODEL_STATE_SUCCESS
+} = actionTypes
 
 const storyCreateMiddleware = () => next => action => {
-  if (action.type === SEND_COMMAND) {
-    if (action.command.type === 'createStory') {
-      if (action.command.ok) {
-        window.location = `${rootDirectory}/storyDetails/${action.aggregateId}`
-      } else if (action.command.error) {
-        window.location = `${rootDirectory}/error?text=Failed to create a story`
-      }
-    }
+  if (
```

Comment:
=>
if (action.commandType === 'createStory') {
window.location = action.type === SEND_COMMAND_SUCCESS ? `${rootDirectory}/storyDetails/${action.aggregateId}` : `${rootDirectory}/error?text=Failed to create a story`
}